app/services/admin/application_csv_generator.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require "csv"
+
+module Admin
+  class ApplicationCsvGenerator
+    CSV_EXPORT_COLUMNS = [
+      :id,
+      :cohort_start_year,
+      :npq_course_id,
+      :npq_lead_provider_id,
+      :npq_course_name,
+      :npq_lead_provider_name,
+      # :teacher_catchment, # Not currently sent to ECF from NPQ
+      # :teacher_catchment_country, # Not currently sent to ECF from NPQ
```

Comment:
These will need uncommenting and including in the export once we get these sending from NPQ into ECF.